### PR TITLE
Update noticeerror-browser-agent-api.mdx

### PR DESCRIPTION
**Pull request by dbabannin-nr**
Provided information on JavaScriptError limit
Related to DEMPLA-2374

### CODE REVIEW

**Comment by barbnewrelic** [comment]
@dbabannin-nr  Just a reminder that we need for you to click the CLA button on your PR's Conversation tab. Signing the Contributor License Agreement is a requirement for all contributors for our docs site. Then we can proceed with reviewing and merging your PR. Thanks again!

File: src/content/docs/browser/new-relic-browser/browser-agent-spa-api/noticeerror-browser-agent-api.mdx [hunk 1]
```diff
@@ -46,6 +46,8 @@ You can also use the API to notice errors that otherwise would be reported witho
 var err = new Error('<var>Report caught error to New Relic</var>');newrelic.noticeError(<var>err</var>);
 ```
 
+Note: If more than 1000 `JavaScriptError` events is sent in a single request, only 1000 separate events is recorded. However, total number of events is preseved internally and accessible by using `EXTRAPOLATE` query clause.
+
```
Comment (obarney) [inline src/content/docs/browser/new-relic-browser/browser-agent-spa-api/noticeerror-browser-agent-api.mdx:50]: I would suggest linking to the docs that explain what EXTRAPOLATE does: https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#extrapolate

Comment (barbnewrelic) [reply src/content/docs/browser/new-relic-browser/browser-agent-spa-api/noticeerror-browser-agent-api.mdx:50]: @obarney We will be happy to incorporate your suggestion into doc update after @dbabannin-nr  signs the Contributor License Agreement button on the Conversation tab of this PR. The CLA is required of all contributors to our docs site.

**Review by barbnewrelic — APPROVED** [review]
Thanks for adding this helpful info to the API doc!

**Pull request merged**